### PR TITLE
setup homepage

### PR DESCRIPTION
**Pull request by ugwulo**
I have resolved the issue with the build exception error, the issue was caused because I used the new <com.google.android.material.navigation.NavigationView> instead of <fragment> in the navhost fragment.

please review @jumaallan @mikailyusuf

### CODE REVIEW

File: app/src/main/java/com/example/gitrank/ui/Explore.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.example.gitrank.databinding.FragmentExploreBinding
+
+
+class Explore : Fragment() {
```
Comment (jumaallan) [inline app/src/main/java/com/example/gitrank/ui/Explore.kt]: I think it's best practise to name the classes well. For example, if we call this `Explore`, someone might not understand it well

We can rename it to `ExploreFragment`

File: app/src/main/java/com/example/gitrank/ui/Repositories.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.example.gitrank.databinding.FragmentRepositoriesBinding
+import com.example.gitrank.databinding.FragmentUsersBinding
+
+
+class Repositories : Fragment() {
```
Comment (jumaallan) [inline app/src/main/java/com/example/gitrank/ui/Repositories.kt]: Call this `RepositoriesFragment`

File: app/src/main/java/com/example/gitrank/ui/Users.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.example.gitrank.databinding.FragmentUsersBinding
+
+
+class Users : Fragment() {
```
Comment (jumaallan) [inline app/src/main/java/com/example/gitrank/ui/Users.kt]: This to `UsersFragment`

**Review by jumaallan — CHANGES_REQUESTED** [review]
Look at the requested changes

File: app/src/main/java/com/example/gitrank/ui/Explore.kt (hunk 1 above)
Comment (ugwulo) [reply app/src/main/java/com/example/gitrank/ui/Explore.kt]: My bad, 
I will sure update them

**Comment by ugwulo** [comment]
I have used the appropriate file names for the Fragments and went on to modify my earlier commit to use the recommended bottom navigation for multiple nav graphs. Reference here -> https://github.com/android/architecture-components-samples
Please review @jumaallan @mikailyusuf 

Screenshot_20210504-085743

**Comment by mikailyusuf** [comment]
LGTM

**Pull request merged**